doc/source/ray-core/tasks.rst

```diff
@@ -259,6 +283,10 @@ You can change this behavior by setting
 in :func:`ray.remote() <ray.remote>` and :meth:`.options() <ray.remote_function.RemoteFunction.options>`.
 See :ref:`Ray fault tolerance <fault-tolerance>` for more details.
 
+.. tip::
+
+    From Ray 2.3 onwards, you could also see the tasks exit details (e.g. OOM killed) and task retries attempts with increasing attempt number using :ref:`Ray's State API <state-api-overview-ref>``  to query the actors information, or go to the :ref:`Dashboard <observability-getting-started>`` Actor page to see more details. (These features all require installing ray with `pip install "ray[default]"`). 
```

Comment:
Can you add this to fault tolerance doc + embed the output to that doc (first vs second retry)?